### PR TITLE
Use a valid SPDX License expression

### PR DESCRIPTION
**Pull request by akshaaatt**
More information is available at https://spdx.org/licenses/

### CODE REVIEW

**Review by yvanzo — COMMENTED** [review]
* Is SPDX identifier expected by some tool you are using? It might be worth mentioning which error does it fix for which tool.
* The letter D of SPDX is missing in your commit’s message (and thus in your pull request’s title too).

**Review by mwiencek — APPROVED** [review]
> Is SPDX identifier expected by some tool you are using? It might be worth mentioning which error does it fix for which tool.

I'm curious about that too, but it seems sensible to follow the standard in any case.

(At the same time, this isn't a real package being published; `"private": true` is set and we're only using it to track dependencies. So it's unclear what files the license field here is even referring to.)

**Comment by yvanzo** [comment]
> I'm curious about that too, but it seems sensible to follow the standard in any case.

Yes

**Comment by akshaaatt** [comment]
Corrections made!

File: package.json [hunk 1]
```diff
@@ -93,5 +93,5 @@
     "utf8": "2.1.2",
     "xgettext-js": "3.0.0"
   },
-  "private": true
+  "private": false
```
Comment (mwiencek) [inline package.json:96]: Sorry for the misunderstanding, but I wasn't requesting for this to be changed.  Please revert this, as this isn't meant to be published.

Comment (akshaaatt) [reply package.json:96]: Fixed in https://github.com/metabrainz/musicbrainz-server/pull/2377

**Pull request merged**